test/bind_params_length_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+require 'models/topic'
+
+class BindParamsLengthTest < Test::Unit::TestCase
+  def setup
+    TopicMigration.up
+    Topic.create!(title: 'blub')
+  end
+
+  def teardown
+    TopicMigration.down
+  end
+
+  def test_bind_param_length
+    assert Topic.where(id: [1] * 75000).first
```

Comment:
One thing to note is that 75000 parameters should work in AR 5.2 before this change, it only fails for parameter sizes between 32767 and 65535.

Reply:
@jessicaspeir Oops...you're right of course...forgot to stage (once  again). I used 75000 initially to have the test passing before the change to verify the test itself was basically ok.

@enebo Should I do another PR or may I ask you to change the value to 35000? (I just replaced that glued in battery in my MacBook and then had the silly idea to update to Mojave...and not everything is working as it should just yet... 😉 )

Reply:
@dr-itz yeah I can change it!  Will do shortly.

Reply:
@jessicaspeir Thanks for catching this.  We appreciate another set of eyes...

Reply:
Changed in fc44fedf

Reply:
No problem, thanks for fixing it so quickly!